vault/README.adoc

```diff
@@ -1,34 +1,46 @@
 # Plugin for Vault and Rudder
 
-This project is part of Rudder - IT infrastructure Automation and Compliance.
-See: http://rudder.io for more information.
+This project is part of Rudder - IT infrastructure Automation and Compliance. See: http://rudder.io for more information.
 
-See repository main README for information about licensing, contribution, and
-other general information.
+See repository main README for information about licensing, contribution, and other general information.
 
 // Everything after this line goes into Rudder documentation
 // ====doc====
 [vault-plugin]
-= Vault
+== Vault
```

Comment:
Should only be one `=` here (It's the second first livel title of the page but we need it because we only include it in the doc starting from the line just before, so it needs to be a first level title.

Reply:
i ve changed it but looks like asciidoctor doesnt validate it, gonna push it anyway